.github/workflows/spelling3.yml

```diff
@@ -59,35 +83,41 @@ jobs:
       contents: read
       pull-requests: read
       actions: read
+      security-events: write
```

Comment:
this is required for sarif reporting

Reply:
I think I have to veto on this permission.  I don't know what feature this removes, but allowing an action outside of our control to write is not going to work.  If they had `create` or similar permission level so that you could create new items but not overwrite existing ones, then it would likely be more acceptable.

Reply:
Yeah, I agree. I'm really annoyed that there isn't such a thing. If you can find someone to poke to get them to make a better api surface, I'd be overjoyed 😉.

Reply:
[community/discussions#29710](https://github.com/orgs/community/discussions/29710)

Closest thing I found.